src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -117,6 +125,31 @@ class PackageDetailTab extends mixin(StoreMixin) {
     }
   }
 
+  componentWillUnmount() {
+    MesosStateStore.removeChangeListener(
+      MESOS_STATE_CHANGE,
+      this.onMesosStateChange
+    );
+  }
+
+  componentWillMount() {
+    MesosStateStore.addChangeListener(
+      MESOS_STATE_CHANGE,
+      this.onMesosStateChange
+    );
+  }
+
+  onMesosStateChange() {
+    this.setState({
+      runningPackageNames: {
+        state: "success",
+        data: (MesosStateStore.getLastMesosState().tasks || [])
+          .filter(_ => _.state === "TASK_RUNNING")
```

Comment:
Not blocking feedback but just thought I would share that this confused me for a sec. Usually i take "_" params to be unused, and "_" operands to be lodash

Reply:
good point. when wrangling data i seemingly have developed a weird style of trying to hide stuff that's not really needed in my head.

we can definitely make that `task => task.state === "TASK_RUNNING"`.